src/ex5/server/services/item_manager.js

```diff
@@ -0,0 +1,93 @@
+// The ItemManager should go here. Remember that you have to export it.
+
+const { Item } = require('../db/models');
+
+const SORT_OPTIONS = ['id', 'text', 'status']
+
+class ItemManager {
+  init() {
+    this.sortOptionIndex = 0;
+    this.sortBy = SORT_OPTIONS[this.sortOptionIndex];
+  }
+
+  async addItem(text) {
+    try {
+      return await Item.create({
+        text: text,
+        isNew: true,
+        status: false,
+        done: null
+      });
+    } catch (error) {
+      return await Item.update({
+        isNew: true,
+      }, {
+        where: { text: text }
+      })
+    }
+  }
+
+  async getItem(itemId) {
+    return await Item.findAll({
+      where: {
+        id: itemId
+      }
+    });
+  }
+
+  async updateItem(itemId, body) {
+    return await Item.update({
+      text: body.text,
+      isNew: body.isNew,
+      status: body.status,
+      done: body.done
+     }, {
+      where: { id: itemId }
```

Comment:
I would suggest adding a data access layer to wrap the access to your DB that way you can have one place where all DB access functions are defined and if you need to add to them / change them later then you can do it just in one place. 

For example if you add a updateItemById function that gets the itemId and updates object then you can do theactual update of the DB there and then if you have to switch the DB or add to it somehow in the future you will have just one place to make the change

Reply:
I will open a new branch for it